DRN.Framework.SharedKernel.Analyzers/AnalyzerReleases.Unshipped.md

```
@@ -0,0 +1,10 @@
+﻿; Unshipped analyzer release
+; https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md
+
+### New Rules
+
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+DRN0001 | DRN.Domain | Error | DiagnosticDescriptors
+DRN0002 | DRN.Domain | Error | DiagnosticDescriptors
+DRN0003 | DRN.Domain | Error | DiagnosticDescriptors
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)